src/electron/start.js

```diff
@@ -12,7 +12,35 @@ const tryConnection = () => client.connect({ port: port }, () => {
         console.log('starting electron');
         startedElectron = true;
         const exec = require('child_process').exec;
-        exec('npm run electron');
+        const electron = exec('npm run electron:run:dev', (error, stdout, stderr) => {
```

Comment:
Thanks for adding this - this should help out when diagnosing weird electron start issues.